src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,12 @@ public CommandResult execute() {
         }
     }
 
+    /**
+     * Command mutalates data, returning true
```

Comment:
A more precise statement would be
"Returns true if .... and false if .. . "
This removes all ambiguity.